db/seeds/de2d6169-c769-4571-b3f2-220bcf8c3ce9/version.json

```diff
@@ -59,7 +59,7 @@
     "work_items": [
       {
         "id": "cf5e303e-98dd-4b0f-97ea-3560c4c5f137",
-        "uplift": 10,
+        "uplift": 0,
```

Comment:
any reason for this?

Reply:
just wanted to be able to test work item uplifts tab where there were no uplifts (therefore no remove all uplifts button)